cpp/lib/include/DbConnection.h

```diff
@@ -153,10 +153,13 @@ class DbConnection {
     /** \note Converts the binary contents of "unescaped_string" into a form that can used as a string.
      *  \note This probably breaks for Sqlite if the string contains binary characters.
      */
-    std::string escapeString(const std::string &unescaped_string, const bool add_quotes = false);
+    std::string escapeString(const std::string &unescaped_string, const bool add_quotes = false, const bool return_null_on_empty_string=false);
```

Comment:
Missing spaces.  Also it might be nice to document this parameter.  I guess it if it is true and we would have o/w returned an empty string we return "NULL" instead?